tests/frequentist_stats/test_sample_size_calc.py

```diff
@@ -19,7 +20,7 @@
 uri_count = ConfigLoader.get_metric("uri_count", "firefox_desktop")
 
 
-@pytest.fixture()
```

Comment:
Was this an automated change?

Reply:
Yeah, I'm not sure how this would change things but the tests that use the fixture still work so 🤷

Reply:
Seems like it's normal: https://docs.pytest.org/en/6.2.x/fixture.html